### PR TITLE
Quiz page added in header.

### PR DESCRIPTION
**Pull request by yashi-15**
### Quiz page is added in navigation bar

closes#1471

### Changes Done:

- Added a link to Quiz page in the navigation bar through "Play Quiz".
- Created a Quiz page which is linked to "Play Quiz".
- Removed the quiz section from home page as now it can be accessed through navigation bar.
- Added some styling for the created page.

### Check list

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
Screenshot (18)

### CODE REVIEW

**Comment by rohanbabbar04** [comment]
I think if we have added the `Play Quiz` on the menu bar, there is no point of having a pop-up for the quiz. So i think we should remove that...

**Comment by yashi-15** [comment]
I have removed pop-up and some repeating code which was there. Please check and let me know if some other changes are required.

File: index.html [hunk 1]
```diff
@@ -1223,7 +1211,7 @@ <h1 class="faq-page">Q: How can I access the Online library? </h1>
     })
 
   </script>
-  <script>
+  <!-- <script>
```
Comment (rohanbabbar04) [inline index.html:1214]: Remove this instead of commenting your code

File: index.html [hunk 2]
```diff
@@ -1234,8 +1222,8 @@ <h1 class="faq-page">Q: How can I access the Online library? </h1>
     }
 
 
-  </script>
-  <script>
+  </script> -->
+  <!-- <script>
```
Comment (rohanbabbar04) [inline index.html:1226]: Same here....

File: index.html [hunk 3]
```diff
@@ -206,7 +206,8 @@
   </nav>-->
 
   <!--modal-->
-  <div class="modal fade" id="myModal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1"
+
+  <!-- <div class="modal fade" id="myModal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1"
```
Comment (rohanbabbar04) [inline index.html:210]: Remove this piece of code instead of commenting it..

File: index.html [hunk 4]
```diff
@@ -832,21 +833,8 @@ <h1 class="title">Get In Touch</h1>
...
-      <!-- ***************** Game Page ***************** -->
-      <a class="quiz-btn" href="/Quiz/pages/game.html" target="_blank">Play</a>
-      <!-- ***************** Highscore Page ***************** -->
-      <a class="quiz-btn" href="/Quiz/pages/highscore.html" target="_blank">High Scores</a>
-
-    </div>
-    <img class="quiz-img" src="/Quiz/brain.webp" alt="">
-  </div>
-  <!-- Quiz section under issue #534  ********* Ends ********** -->
+<!-- Removed Quiz section from Home page and added added a new quiz page.   -->
```
Comment (rohanbabbar04) [inline index.html:837]: Good :)

File: Quiz/css/style.css [hunk 5]
```diff
@@ -1,17 +1,26 @@
   
   /* These will be used for all pages  */
   
+  body {
+    width: 100%;
+    height: 100vh;
+    font-family: 'Poppins', sans-serif;
+    background-image: url(https://img-cdn.inc.com/image/upload/images/panoramic/GettyImages-577674005_492115_zfpgiw.webp);
+
```
Comment (rohanbabbar04) [inline Quiz/css/style.css:9]: Remove this extra line...

**Review by rohanbabbar04 — COMMENTED** [review]
Currently the `Play` and `High Score` direct the user to a new tab, that should'nt be the case. As for convenience, we want them to stay on the same tab.

**Comment by yashi-15** [comment]
### Changes done:
- Extra code is removed.
- Play and High score is not opening in new tab now.

 Please let me know if any further changes needed. Thank You!  @rohanbabbar04

File: Quiz/pages/highscore.html [hunk 6]
```diff
@@ -14,7 +14,7 @@
 <body>
   <div class="quiz-container">
     <div id="highScores" class="flex-center flex-column">
-      <h1 id="finalScore">High Scores</h1>
+      <h1 id="finalScore" style="color: black;">High Scores</h1>
       <ul id="highScoresList"></ul>
       <a class="quiz-btn" href="./index.html">Go Home</a>
```
Comment (rohanbabbar04) [inline Quiz/pages/highscore.html:19]: Lets fix this as well 
Should be `../../index.html`

Comment (yashi-15) [reply Quiz/pages/highscore.html]: Fixed!

**Comment by rohanbabbar04** [comment]
Looks great thanks :)

**Review by rohanbabbar04 — APPROVED** [review]

**Pull request merged**